ui/src/components/MainContainer.js

```diff
@@ -210,7 +210,7 @@ const MainContainer = () => {
         <PrivateRoute
           exact
           path="/admin/:pageName"
-          permissions={['admin', 'stewardship']}
+          permissions={['admin', 'stewardship', 't3ch']}
```

Comment:
Hmm, I don't think this is the best implementation for admin pages. As of right now it means we can't permission each admin pages individually (which we want to)